components/description/description.module.scss

```diff
@@ -0,0 +1,78 @@
+@import "../../styles//media.scss";
+
+
+.container {
+  background: #fff;
+  height: 371px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0px 0px 5px 1px rgb(0,0,0,0.3);
+  position: relative;
+  @media only screen and (max-width: $mobile) {
+    height: 409px;
+    flex-direction: column;
+    overflow: inherit;
+    // justify-content: center;
+    // align-items: center;
+  }
+}
+
+.content {
+  margin-right: 143px;
+  @media only screen and (max-width: $mobile) {
+    padding: 24px 16px;
+    margin-right: 0;
+  }
+
+  @media only screen and (max-width: $tablet) {
+    padding: 24px 16px;
+    margin-right: 0;
+  }
+}
+
+.title {
+  font-size: 36px;
+  font-weight: 500
+  ;
+
+  @media only screen and (max-width: $mobile) {
+    font-size:24px;
+  }
+
+  @media only screen and (max-width: $tablet) {
+    font-size:24ppx;
+  }
+}
+
+.description {
+  font-size: 16px;
+  margin-top: 15px;
+
+  @media only screen and (max-width: $mobile) {
+    font-size:14px;
+  }
+
+}
+
+
+.contributions {
+  position: relative;
+  left: 5%;
+
+  @media only screen and (max-width: $mobile) {
+  position: absolute;
+    padding: 0;
+    top: 70%;
+    left: 50% !important;
```

Comment:
Please do not use important.
You can fix this issue and most of the issues with working with grid, if you not familiar with grid, I am recommended to see this lecture by Elad Shechter: https://www.youtube.com/watch?v=nor6oxitW54, and use this article by Cris Coyer: https://css-tricks.com/snippets/css/complete-guide-grid/.

Hint, do not use for this grid-template-areas.

If you need help, I feel free to talk to me :)